packages/buidler-core/src/builtin-tasks/compile.ts

```diff
@@ -181,7 +172,7 @@ export default function () {
 
           const emittedArtifacts = [];
           for (const [contractName, contractOutput] of Object.entries(
-            output.contracts[file.globalName]
+            output.contracts?.[file.globalName] ?? {}
```

Comment:
This fixes a weird bug I found by coincidence. Basically, if you had an `Importer.sol` with only import statements but no contracts, then this line would throw (because `Object.entries(undefined)` throws).

Reply:
Good catch!